src/external-types/reveal.js/index.d.ts

```diff
@@ -4,9 +4,199 @@
+    // like top/right/bottom/left, width/height or margin.
+    autoAnimateStyles?: AutoAnimateStyles[],
+    // Should unmatched elements get auto-animate transitions
+    autoAnimateUnmatched?: boolean, // ToDo: If we omit autoAnimateMatcher, we don't need this do we?
+    // Global override for autolaying embedded media (video/audio/iframe)
+    // - null:   Media will only autoplay if data-autoplay is present
+    // - true:   All media will autoplay, regardless of individual setting
+    // - false:  No media will autoplay, regardless of individual setting
+    autoPlayMedia?: boolean | null,
+    // Controls automatic progression to the next slide
+    // - 0:      Auto-sliding only happens if the data-autoslide HTML attribute
+    //           is present on the current slide or fragment
+    // - 1+:     All slides will progress automatically at the given interval
+    // - false:  No auto-sliding, even if data-autoslide is present
+    autoSlide?: 0 | 1 | false, // 1:
+    // Use this method for navigation when auto-sliding (defaults to navigateNext)
+    /* autoSlideMethod: null, // ToDo: I think this is not needed */
+    // Stop auto-sliding after user input
+    autoSlideStoppable?: boolean,
+    // Transition style for full page slide backgrounds
+    backgroundTransition?: Transition,
+    // Vertical centering of slides
+    center?: boolean,
+    // Display presentation control arrows
+    controls?: boolean,
+    // Visibility rule for backwards navigation arrows
+    controlsBackArrows?: ControlsBackArrows,
+    // Determines where controls appear
+    controlsLayout?: ControlsLayout,
+    // Help the user learn the controls by providing hints, for example by
+    // bouncing the down arrow when they first encounter a vertical slide
+    controlsTutorial?: boolean,
+    // Specify the average time in seconds that you think you will spend
+    // presenting each slide. This is used to show a pacing timer in the
+    // speaker view
+    defaultTiming?: number | null,
+    /* disableLayout?: boolean, // ToDo: I think this is not needed */
+    // The display mode that will be used to show slides
+    display?: Property.Display,
+    // Flags if the presentation is running in an embedded mode,
+    // i.e. contained within a limited portion of the screen
+    embedded?: boolean,
+    // Focuses body when page changes visibility to ensure keyboard shortcuts work
+    focusBodyOnPageVisibilityChange?: boolean,
+    // Flags whether to include the current fragment in the URL,
+    // so that reloading brings you to the same fragment position
+    /* fragmentInURL?: boolean, // ToDo: I think this is not needed */
+    // Turns fragments on and off globally
+    fragments?: boolean,
+    // Add the current slide number to the URL hash so that reloading the
+    // page/copying the URL will return you to the same slide
+    /* hash?: boolean, // ToDo: I think this is not needed */
```

Comment:
> Add the current slide number to the URL hash so that reloading the
> page/copying the URL will return you to the same slide

I think this also depends on, if we want `fragmentInURL`…

Reply:
That's more or less the same thing as above.